mocha.el

```diff
@@ -193,12 +193,8 @@ If we reach the root without finding what we are looking for return nil."
 When a 'describe' or 'it' is found, return the first argument of that call.
 If js2-mode is not enabled in the buffer, returns nil.
 If there is no wrapping 'describe' or 'it' found, return nil."
-  (if (string= major-mode "js2-mode")
-      (let ((node (js2-node-at-point)))
-        (mocha-walk-up-to-it node))
-    (progn
-      (message "js2-mode must be enabled to run test at point.")
```

Comment:
So the cucumber test is failing because now this line throws an error, instead of just printing a message, when you are not using any derivative of js2-mode.

I know we discussed removing this message, so we have a couple options.

1. Put the message and condition back to make the tests happy.
2. Make a new test step here https://github.com/scottaj/mocha.el/blob/master/features/step-definitions/mocha-steps.el. Something like (When "^I run the command \"\\([^\"]+\\)\" expecting an error$" that will run the command and catch any thrown errors. IDK how easy that would be to write off the top of my head.

I'm fine with either solution.